tests/AbstractProcessTest.php

```diff
@@ -299,6 +317,49 @@ public function testTerminateWithStopAndContinueSignalsUsingEventLoop()
     }
 
     /**
+     * @dataProvider provideOutputSizeAndExpectedMaxDuration
+     */
+    public function testProcessWithFixedOutputSize($size, $expectedMaxDuration = 5)
+    {
+        // Note: very strange behaviour of Windows (PHP 5.5.6):
+        // on a 1000 long string, Windows succeeds.
+        // on a 10000 long string, Windows fails to output anything.
+        // On a 100000 long string, it takes a lot of time but succeeds.
```

Comment:
@moufmouf: Per your explanation in #10 and the above comments, this test fails on Windows. Does the workaround you're proposing in #11 fix this?